README.md

```diff
@@ -305,7 +305,26 @@ Each directory will be created with `cl-keytool.jar`, `cl-wallet.jar`, and a P12
 **In `code/data-l1`:**
 -   data-l1.jar     // The executable for the dL1 layer
 
+#### Parameters
+If you run the command `./hydra remote-deploy -h` you should see something like
 
+```
+Remotely deploy to cloud instances using Ansible
+
+USAGE: hydra remote-deploy [OPTIONS]
+
+OPTIONS:
+      --force_genesis                                  Force metagraph to deploy as genesis
+      --owner_p12_file_name <OWNER-P12-FILE-NAME>      Metagraph owner p12 file name (File to deduct fees). Be sure to have the p12 file under `source/p12-files` directory
+      --staking_p12_file_name <STAKING-P12-FILE-NAME>  Metagraph staking p12 file name (File to reduce fees based on balance). Be sure to have the p12 file under `source/p12-files` directory
+  -h, --help   
+```
+* --force_genesis: Use this flag when you want your metagraph to start from genesis. If you have already started a metagraph and need to execute from genesis again, use this command.
```

Comment:
We should add a warning here that this will erase all existing snapshot state from the nodes.

Reply:
What do you think? Do we have another warning so we don't need this?

Reply:
I'll add in README also, but I've added in the remote functions already